paper.tex

```diff
@@ -1895,6 +1903,58 @@ \subsection{AR Art Gallery}
 	\label{fig:exhibitionScan}
 \end{figure}
 
+\subsection{Alterniative Uses for AR in ARuko}
+\label{sec:alternativeARukoAR}
+
+Even once the idea for ARuko was settled on audio tour functionality with 
+additional AR components, what exactly those AR components would do was still
+undecided for a while. Atticus knew he wanted to use AR on the tour, but wasn't
```

Comment:
"undecided until more details about the project crystallized," also change "wasn't" to "was not"